plugin/src/main/java/org/jenkinsci/plugins/structs/describable/UninstantiatedDescribable.java

```diff
@@ -175,8 +188,12 @@ public Object instantiate() throws Exception {
      *      depends on this parameter.
      */
     public <T> T instantiate(Class<T> base) throws Exception {
+        return instantiate(base, null);
+    }
+
+    public <T> T instantiate(Class<T> base, TaskListener listener) throws Exception {
```

Comment:
Looks like you need to move the Javadoc here and deprecate the old method.

Reply:
blah, good catch